troposphere/static/js/components/common/breadcrumb/Breadcrumb.react.js

```diff
@@ -23,13 +23,10 @@ define(function(require) {
 
     renderLink: function() {
       var class_names = this.props.breadcrumb.state;
-      var divStyle = {
-        width: this.props.width + '%'
-      };
 
       return (
-          <div style={divStyle} onMouseOver={this.mouseOver} onMouseOut={this.mouseOut} className={class_names} onClick={this.crumbClicked}>
-            {this.props.breadcrumbText}
+          <div onMouseOver={this.mouseOver} onMouseOut={this.mouseOut} className={class_names} onClick={this.crumbClicked}>
+            <span className="hidden-xs">Step </span> {this.props.breadcrumbText}
```

Comment:
I thought about that, I'm torn though. In the situation where we have seven steps, unless we make the modal wider, I don't think some steps like "Version and Provider" will fit well. 

I was thinking this might be something we address in the redesign? Like, do we want to do these things in a modal in the first place?